### PR TITLE
Design Changes

### LINKED ISSUE #18: Codewise looks good, but one note on behaviour:
Codewise looks good, but one note on behaviour:

I would expect some indication when the button is clicked. Like border changing a the color a bit.
Really good would be a ripple effect, but I don't know how difficult that is. (Like what you can see at the examples here: https://vuetifyjs.com/en/components/buttons/#usage)

_Originally posted by @Taucher2003 in https://github.com/code0-tech/base-ui/pull/14#pullrequestreview-1785304817_

### PR DESCRIPTION
**Pull request by nicosammito**
Close #18

### CODE REVIEW

File: src/styles/_helpers.scss [hunk 1]
```diff
@@ -32,11 +25,45 @@
...
-  &--active {
+  &:active, &--active {
     @content
   }
 }
 
-@function borderColor($hover: false) {
-  @return rgba($white, if($hover, 0.2, 0.1))
+@function borderColor($interacted: false) {
+  @return rgba($white, if($interacted == true, .2, .1))
```
Comment (Taucher2003) [inline src/styles/_helpers.scss:34]: Does this need `== true`?
```suggestion
  @return rgba($white, if($interacted, .2, .1))
```

File: src/styles/_helpers.scss [hunk 2]
```diff
@@ -32,11 +25,45 @@
...
+}
+
+@mixin border($interact: true) {
+
+  border: 1px solid borderColor();
+
+  @if ($interact == true) {
```
Comment (Taucher2003) [inline src/styles/_helpers.scss:41]: Does this need `== true`?

```suggestion
  @if ($interact) {
```

**Review by Taucher2003 — DISMISSED** [review]

**Comment by Taucher2003** [comment]
Small side note: It seems like there is a bit more space at the bottom than at the top of the alert without body

image

**Comment by nicosammito** [comment]
> Small side note: It seems like there is a bit more space at the bottom than at the top of the alert without body
> 
> image

fixed with [72a694d](https://github.com/code0-tech/pictor/pull/26/commits/72a694d696ddade2c698d4a562e8e266acd1335d)

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**